pkg/channel/fanout/fanout_handler_test.go

```diff
@@ -40,11 +40,11 @@ const (
 )
 
 func makeCloudEvent() cloudevents.Event {
-	event := cloudevents.NewEvent(cloudevents.VersionV03)
+	event := cloudevents.NewEvent(cloudevents.VersionV1)
 	event.SetType("com.example.someevent")
 	event.SetSource("/mycontext")
 	event.SetID("A234-1234-1234")
-	event.SetExtension("comExampleExtension", "value")
+	event.SetExtension("comexampleextension", "value")
```

Comment:
just jotting this down here, but 0.3 used to default encoding, and now you have to manually set it, might want to add that in here as well?